index.html

```diff
@@ -20,24 +20,25 @@
         </li>
       </ul>
     </div>
-    <br>
+    <br />
     <section class="header-container">
       <div class="sub-container">
         <h1 class="header-highlight-1">Hey There.I’m OMSOO</h1>
```

Comment:
- [ ] Kindly, stick to the design as much as possible for better UI (user interface) and this is a good professional practice that every frontend software developer should maintain. Please, make sure to make the hero title closely similar to the [original design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10).

**Current mobile menu**           |  **Original design**
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/80895497/153454521-840b33a5-400b-4de0-a1b8-7f9d756f855c.PNG)  |  ![](https://user-images.githubusercontent.com/80895497/153454526-05207c76-a132-4546-9773-573036d4ca21.PNG)